ts/screens/ingress/IngressScreen.tsx

```diff
@@ -1,104 +1,63 @@
-    const items = [
-      {
-        enabled: this.props.hasSessionToken,
-        label: I18n.t("startup.authentication")
-      },
-      {
-        enabled: this.props.hasSessionInfo,
-        label: I18n.t("startup.sessionInfo")
-      },
-      { enabled: this.props.hasProfile, label: I18n.t("startup.profileInfo") },
-      {
-        enabled: this.props.isProfileEnabled,
-        label: I18n.t("startup.profileEnabled")
-      }
-    ];
-    return (
-      <SafeAreaView
-        style={[IOStyles.flex, { backgroundColor: variables.brandPrimary }]}
-      >
-        <StatusBar
-          barStyle="light-content"
-          backgroundColor={styles.container.backgroundColor}
-        />
-        <Container style={styles.container}>
-          <View style={IOStyles.alignCenter}>
-            <Body color="white">{I18n.t("startup.title")}</Body>
-          </View>
-          <Spinner color="white" />
-
-          <List withContentLateralPadding={true}>
-            {items.map((item, index) => (
-              <ListItem key={`item-${index}`}>
-                <IngressCheckBox checked={item.enabled} />
-                <HSpacer size={8} />
-                <Body
-                  color="white"
-                  weight={item.enabled ? "SemiBold" : "Regular"}
-                >
-                  {item.label}
-                </Body>
-              </ListItem>
-            ))}
-          </List>
-          <View style={{ marginTop: 48 }}>
-            <SectionStatusComponent sectionKey={"ingress"} />
-          </View>
-        </Container>
-      </SafeAreaView>
-    );
-  }
-}
+const SPINNER_SIZE = 48;
+const SPACE_BETWEEN_SPINNER_AND_TEXT = 24;
 
-function mapStateToProps(state: GlobalState) {
-  const maybeSessionToken = sessionTokenSelector(state);
-  const maybeSessionInfo = sessionInfoSelector(state);
-  const potProfile = profileSelector(state);
-  return {
-    hasSessionToken: maybeSessionToken !== undefined,
-    hasSessionInfo: O.isSome(maybeSessionInfo),
-    hasProfile: potProfile !== null,
-    isProfileEnabled:
-      pot.isSome(potProfile) &&
-      potProfile.value.has_profile &&
-      potProfile.value.is_inbox_enabled
-  };
-}
+export const IngressScreen = () => {
+  const contentTitle = I18n.t("startup.title");
+  useOnFirstRender(() => {
+    trackIngressScreen();
+    // Since the screen is shown for a very short time,
+    // we prefer to announce the content to the screen reader,
+    // instead of focusing the first element.
+    if (Platform.OS === "android") {
+      // We use it only on Android, because on iOS the screen reader
+      // stops reading the content when the ingress screen is unmounted
+      // and the focus is moved to another element.
+      AccessibilityInfo.announceForAccessibility(contentTitle);
+    }
+  });
 
-export default connect(mapStateToProps)(IngressScreen);
+  return (
+    <SafeAreaView style={styles.container}>
+      <ContentWrapper>
+        <View style={{ alignItems: "center" }}>
```

Comment:
No inline style?

Reply:
When is only a single style value I put it directly without using an inline style. Anyway I addressed it with an inline style here f7e95f836a2f448be9a82f5019e577b11e25d30c